main.go

```diff
@@ -104,12 +104,28 @@ var generateCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 
 		if importPath != "" {
-			err := importData(importPath, &file, cmd.Flags())
+			f, err := os.Open(importPath)
 			if err != nil {
 				return err
 			}
+			defer f.Close()
+
+			if strings.HasSuffix(importPath, ".json") {
+				if err = json.NewDecoder(f).Decode(&file); err != nil {
+					return err
+				}
+			} else if strings.HasSuffix(importPath, ".yaml") && strings.HasSuffix(importPath, ".yml") {
```

Comment:
This line requires both the `.yml` tag and the `.yaml` tag.
It should be `or`

Reply:
Good catch @hopefulTex!